.github/workflows/sphinx-build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.10.0
```

Comment:
Any reason to force specify `3.11.0`? `3.11` seems specific enough.

Reply:
Legit, because it was 3.10.0, and before it was 3.9.0. 

From the docs, having only 3.x would be "faster".

Reply:
I used bad syntax there. 

I meant 3.SOMETHING without the 3rd part of the version number.

Reply:
Gotcha. That is how duckbot specifies [3.8](https://github.com/duck-dynasty/duckbot/blob/main/.github/actions/setup-python-venv/action.yml#L26). It is the way to go, imo.